src/main/java/entities/Course.java

```diff
@@ -8,23 +8,28 @@
     private final char session;
     private final ArrayList<Section> lectures;
     private final ArrayList<Section> tutorials;
+    private final ArrayList<String> exclusions;
```

Comment:
Is it storing course IDs? I'd include that in the variable name so it's clear how the exclusion is stored